packages/react/src/auto/polaris/inputs/PolarisDateTimePicker.tsx

```diff
@@ -106,8 +105,8 @@ export const PolarisDateTimePicker = (props: {
       >
         <div style={{ padding: "16px" }}>
           <DatePicker
-            month={getDateTimeObjectFromDate(zonedTimeToUtc(new Date(fieldProps.value), localTz)).month}
-            year={getDateTimeObjectFromDate(zonedTimeToUtc(new Date(fieldProps.value), localTz)).year}
```

Comment:
Previously, it assumed the `fieldProps.value` must have a value. However, if the input has not been modified or has no default (e.g. when creating a record), `fieldProps.value` is actually `null`, making a date an "Invalid Date" and not showing the calendar.